### PR TITLE
Doc changes for accept branch: run session api change

### PR DESCRIPTION
**Pull request by lemunozm**

### CODE REVIEW

**Comment by richiware** [comment]
Depends on eProsima/micro-RTPS-client#12

File: docs/client.rst [hunk 1]
```diff
@@ -259,10 +290,10 @@ This implies:
 
 The ``_until_timeout`` suffix function version will perform these actions until the waiting for a new message reaches the timeout.
```
Comment (richiware) [inline docs/client.rst]: Not clear for the user this function finishes when a user data was received. Clarify.

Comment (lemunozm) [reply docs/client.rst]: Updated

File: docs/client.rst [hunk 2]
```diff
@@ -265,14 +265,15 @@ This implies:
...
-3. Listens messages from the agent and call the associated callback (a topic callback or a status callback).
+3. Listens messages from the agent and call the associated callback if exists (a topic callback or a status callback).
 
-The ``time`` suffix function version will perform these actions and will listen messages for a `time` duration.
+The ``time`` suffix function version will perform these actions and will listen messages for a ``time`` duration.
+Only when the time waiting for a message overcome the ``time`` duration, the function finished.
 The function will return ``true`` if the sent data have been confirmed, ``false`` otherwise.
 
 :session: Session structure previously initialized.
 :timeout: Time for waiting, in milliseconds.
```
Comment (richiware) [inline docs/client.rst]: the parameter name is `time`, not `timeout`

File: docs/client.rst [hunk 3]
```diff
@@ -286,15 +287,14 @@ This implies:
 
 1. Flashes all output streams sending the data through the transport.
 2. If there is any reliable stream, it will perform the asociated reliable behaviour to ensure the communication.
-3. Listens messages from the agent and call the associated callback (a topic callback or a status callback).
+3. Listens messages from the agent and call the associated callback if exists (a topic callback or a status callback).
 
-The ``_until_timeout`` suffix function version will perform these actions until the waiting for a new message reaches the timeout.
-Only if the time waiting for a message overcome the timeout, the function finished.
-The function will return ``true`` if has been listened a message, ``false`` if the timeout has been reached.
+The ``_until_timeout`` suffix function version will perform these actions until receiving one message.
```
Comment (richiware) [inline docs/client.rst]: Can you specify it finish when a user's data/message is received?

File: docs/client.rst (hunk 2 above)
Comment (lemunozm) [reply docs/client.rst]: Done!

File: docs/client.rst (hunk 3 above)
Comment (lemunozm) [reply docs/client.rst]: Done!

**Pull request merged**